### DISCUSSION TITLE
機器了解人類情緒

### DISCUSSION THREAD

**Discussion by Raiden1121** [1101-GS4719-C/Python-Community-GS4719-C#2066] [General]
https://pansci.asia/archives/328158

機器解讀人類情是緒透過整合語音（如說話的音高、語調）、文字、臉部表情等資訊，透過深度神經網絡分析，進行情緒的判讀。這項技術的應用非常廣泛，像是服務業可以更好的為客人服務，又或者是在醫療方面，協助瘖啞人士表達感受等......。然而，我認為這項技術還有很多疑慮的部分，像是情緒這種抽象又主觀的現象，很容易有出錯，若是在醫療產業應用，容易造成不確定的危險，因此在實際運用之前，還需要突破層層關卡。

**Comment by JoannaLai909** [comment]
最近做專題我也了解到了「情緒分析」這個詞，才知道原來現在已經有模型是可以透過閱讀文本來計算情感分數並分類文本屬於正向、負向還是中立，並且把情感分數的結果納入分析數據的其中一個要素中，作為一個參考數據，例如分析新聞情感，探討正面新聞是否會讓一間公司股票上漲，而負向新聞是否又會讓公司股價下跌

**Comment by hunteryang0108** [comment]
人類在解讀其他人的情緒上都非常有可能出錯了，更遑論是一個還未成熟發展的機器。話是這樣說，但我還是蠻支持這方面的技術發展，雖然這項技術並不能作為決斷某些事的關鍵，但在於輔助人類執行某些業務上，能提供莫大的幫助。

**Comment by CtyceT** [comment]
感覺機器學習人類情緒的想法非常有趣。過去很難製作的功能，例如測謊儀，可以藉此進步使其更加準確。對於輔助的功能性非常強大
